jwst/master_background/master_background_mos_step.py

```diff
@@ -152,6 +164,8 @@ def process(self, data):
             if self.save_background:
                 self.save_model(master_background, suffix='masterbg1d', force=True)
                 self.save_model(mb_multislit, suffix='masterbg2d', force=True)
+                if bkg_x1d_spectra is not None:
+                    self.save_model(bkg_x1d_spectra, suffix='bkgx1d', force=True)
```

Comment:
We should probably add this new output to the regtest for this module (test_nirspec_masterbackground.py::test_nirspec_mos_mbkg).  I can help put the truth data up on artifactory, if you add it in.

Reply:
I think I've added this correctly to the list of files to check in test_nirspec_masterbackground.py::test_nirspec_mos_mbkg now.

Reply:
Looks right!  I ran the test locally and added the file to the truth set on artifactory.